APE0.rst

```diff
@@ -296,15 +300,18 @@ The procedure for voting to add new Voting Members is:
 * The Coordination Committee and Ombudsperson receive each nomination, check
   that it is factually accurate, that the nominated person accepts the
   nomination, and that their record of community activity adheres to the Code of
-  Conduct.
-* Once a nomination is accepted by the Coordination Committee, the Coordination
-  Committee will put it to a vote among active Voting Members. Votes for adding
-  voting members will be carried out at most twice a year to avoid too many
-  elections. The voting dates will be announced with at least four weeks notice,
+  Conduct. If the candidate accepts, they should provide a brief statement in
+  support of their nomination.
+* Once a nomination is accepted by the Coordination Committee, it will be put it
+  to a vote among active Voting Members. Votes for adding voting members will be
+  carried out at most twice a year to avoid too many elections. The returns
+  officer for the voting member election can be a voting member, and those
+  officer are entitled to vote in the election.
+* The voting dates will be announced with at least four weeks notice,
   and additional nominations will be accepted until two weeks before the
   election date. The voting will be open for at least two weeks. To successfully
   gain an appointment as a Voting Member, the nominee must receive at least four
-  positive votes, as long as that constitutes a majority of those voting.
+  positive votes, as long as that constitutes a majority of those voting for that candidate.
```

Comment:
Why is this? Is it mandatory to select either of the 3 options for all cases to produce a valid ballot (e.g. "abstain" is set as default), or basically two ways of abstinence are possible, only one of which is counted into the election?

Reply:
I am trying to remember the details, it's been a while. We wanted people to be able to abstain from voting on individuals, but I think there was some limitation with Helios where we needed to add an abstain button so that this all worked properly. Maybe @mwcraig can remember the details.

From the perspective of APE 0 this is just clarifying that you are allowed to abstain from an individual candidate and not have to abstain from the whole election with multiple candidates. i.e. if there are 20 voting members and 15 of them voted in the election, but 5 abstained in your vote then you would only need a majority of the 10 who voted to vote yay for you to be elected.

Reply:
>  if there are 20 voting members and 15 of them voted in the election, but 5 abstained in your vote then you would only need a majority of the 10 who voted to vote yay for you to be elected.

and that I think is wrong. in the corner case there are 15 who vote in an election but all but one abstain for a candidate, that candidate should not be selected, the majority of the 15 should be what matters.

Reply:
I disagree. If you think about it like this: why should an election with only one candidate be any different from one with 10?

If it wasn't for the admin we could elect voting members one at a time when they are nominated, one per election. In my eyes we should allow as much granularity as possible in how the elections run, if people want to abstain from a candidate they should be allowed to, much in the same way they could abstain from a whole election with one candidate in it.

Reply:
> in the corner case there are 15 who vote in an election but all but one abstain for a candidate, that candidate should not be selected, the majority of the 15 should be what matters.

I'd argue that the problem in this sort of case would be with the 14 who choose to abstain; folks would need understand that "abstain" is not "no"; it is, instead, a "count me out for this one".

I can dig up the language we have in conda-forge, which I think also specifies a minimum yes vote required for election.

Reply:
re both of the above: "abstain" is an "I don't care, either way" in my book, but with the caveat of "if others are supporting the idea", and yes, it's stronger than not even picking up the ballot paper. A minimum number of yes votes may be a good idea.

Reply:
The wording here is you have to have at least 4 votes and those have to be a majority of those who voted (either yes or no) for that candidate.

Reply:
Ah, right. the full clause is "To successfully gain an appointment as a Voting Member, the nominee must receive at least four  positive votes, as long as that constitutes a majority of those voting for that candidate." 

The intent of this change was to avoid the situation where a candidate receives 4 yes votes and 16 no votes -- in the original wording that person would be appointed. 

I think we included an abstain option on the ballot so that we could make every question on the ballot required. 

I see a couple options here:

1. Leave as-is, since there is a minimum number of required yes votes (so a 1 yes, 16 abstain would fail).
2. Change the minimum of yes votes (would require a new APE 0 amendment, which I'm happy to propose)
3. Impose a minimum percent of voting members for a quorum. (this is what conda-forge does)

Regarding 3:Z conda-forge has had several problems meeting that quorum, so there are a couple of sort-pf-complicated ways to skirt the quorum (see https://conda-forge.org/docs/orga/governance.html#voting and scroll down to the section about quorum).

IIRC, we went with an absolute number here just in case apathy sets in. It could be changed to, for example, X number or Y% of voting members, whichever is higher. In that case I'd propose something like X=4 and Y=10.

Reply:
Re 3: this is what PSF does, too, at least 1/3 of all vote eligible members should cast a vote for an election to be valid, otherwise it's a void one. I would be very much for having policies that follow more closely of PSF, e.g. just as we originally proposed. mostly covered in section 3.9: https://www.python.org/psf/bylaws/

Reply:
e.g. the 4 is already <10% of voting members, and that percentage will keep shrinking more.